src/_nebari/upgrade.py

```diff
@@ -329,11 +342,11 @@ def replace_image_tag(s: str, new_version: str, config_path: str) -> str:
             if current_tag == new_version:
                 return s
             loc = f"{config_path}: {image_name}"
-            response = Prompt.ask(
-                f"\nDo you want to replace current tag [green]{current_tag}[/green] with [green]{new_version}[/green] for:\n[purple]{loc}[/purple]? [Y/n] ",
-                default="Y",
+            response = attempt_fixes or Confirm.ask(
```

Comment:
I thought this would still prompt the user for confirmation but it seems that if `attempt_fixes` is `True`, the right hand side of the statement won't even be executed, right?

Reply:
>  if attempt_fixes is True, the right hand side of the statement won't even be executed, right?

Correct

Reply:
This is exactly what we need, thanks. I was just wondering if it would still try to prompt the user but that does not seem to be the case